sdk/search/CONTRIBUTING.md

```diff
@@ -30,10 +30,7 @@ exercise any new features that you've added.
 ### Frameworks
 We use [nUnit 3](https://github.com/nunit/docs/wiki) as our testing framework.
```

Comment:
nit:  Not your change, but the product name is `NUnit`.   Should we fix this while we're making changes?